### DISCUSSION TITLE
hey friends why when im updating all inside comfy manager i get "send error: Cannot write to closing transport" many times? thanks.

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [Comfy-Org/ComfyUI-Manager#625] [General]
something like send error: Cannot write to closing transport and with aiohttp

**Comment by ltdrdata** [comment]
When performing an update all, ComfyUI-Manager maintains an occupied state until all operations are completed.
During this time, if ComfyUI server try to send messages to the browser side while running a workflow or using extension features, such a phenomenon may occur.
